warehouse/query-core/src/test/java/datawave/query/tables/ShardQueryLogicTest.java

```diff
@@ -219,197 +209,209 @@ protected void runTestQuery(Set<Set<String>> expected, String querystr, Date sta
 
     @Test
     public void testFieldMappingTransformViaProfile() throws Exception {
-        Map<String,String> extraParameters = new HashMap<>();
-        extraParameters.put("include.grouping.context", "false");
-        extraParameters.put("query.profile", "copyFieldEventQuery");
-
-        Date startDate = format.parse("20091231");
-        Date endDate = format.parse("20150101");
-
-        String queryString = "UUID =~ '^[CS].*'";
+        givenQuery("UUID =~ '^[CS].*'");
```

Comment:
Very pretty refactoring.  I like the change a lot, but in future MRs can we keep this kind of refactoring separate from adding functionality?  Hard to validate the validate while going on side quests.

Reply:
Fair comment, I'll keep that in mind for future MRs.